src/test/descriptor_tests.cpp

```diff
@@ -312,6 +353,9 @@ void DoCheck(const std::string& prv, const std::string& pub, const std::string&
 
                 /* Infer a descriptor from the generated script, and verify its solvability and that it roundtrips. */
                 auto inferred = InferDescriptor(spks[n], script_provider);
+                if (ref.size() == 1) {
```

Comment:
In dcc11864a8f7141298d71e5a61e686ccf0032b17 "Test roundtripping of descriptors ignoring keys"

How come this is only checked for non-ranged descriptors?

Reply:
It's only checked for non-combo descriptors (where it obviously doesn't hold...).